csrankings-x.csv

```diff
@@ -309,6 +309,7 @@ Xinyu Wang 0006,University of Michigan,https://web.eecs.umich.edu/~xwangsd,mvasb
 Xinyu Xing,Pennsylvania State University,https://ist.psu.edu/directory/faculty/uxx16,WBN9c6kAAAAJ
 Xinyu Zhang 0003,Univ. of California - San Diego,http://xyzhang.ucsd.edu,HQB8W_IAAAAJ
 Xinyuan Wang,George Mason University,http://cs.gmu.edu/~xwangc,8lzslzgAAAAJ
+Xingyu Zhou 0001,Wayne State University,https://xingyuzhou.org/,AsTyRmwAAAAJ

```

Comment:
EE, not CS. Please clarify why they meet the inclusion criteria.

Reply:
I am sorry for the late reply.

I will check the criteria again. I remembered the faculties who can supervise PhD candidates majored in CS count. The Chair of our department has approved to submit this request.

If this is not the case, please feel free to skip my request.

Thanks.

Reply:
Please confirm that they can _solely_ supervise PhD students for a degree _in the CS department._

Reply:
Bump @h312h

Reply:
He can not solely supervise PhD in CS. Please ignore my adding request. Thanks.